cmd/bridge/main.go

```diff
@@ -7,25 +7,21 @@ import (
 	"flag"
 	"fmt"
 	"io/ioutil"
+	"k8s.io/component-base/logs"
```

Comment:
Curious what `k8s.io/component-base/logs` does.  We use `klog` in the operator, but don't have this dependency.

Reply:
@benjaminapetersen - I tried not to use this but it is in the console-operator in `cmd/console/main.go` and it is in other operators. This contains the `InitLogs` function and several other helper functions for the logs. I started to not use it initially but found myself having to re-write some of the functions already in there.

Reply:
ok works for me